doc/nrf/device_guides/working_with_nrf/nrf54l/peripheral_sensor_node_shield.rst

```diff
@@ -0,0 +1,88 @@
+It is a testing shield with following sensors:
+
+   * ``ADXL362`` - Accelerometer (SPI interface)
+   * ``BMI270`` - Accelerometer and gyroscope (SPI interface)
+   * ``BME688`` - Temperature, pressure, humidity and gas sensor (I2C interface with on-board pull-up resistors 4.7 kohm).
+                  It is targeted for testing I2C, and SPI protocols drivers.
+
+Supply voltage - 1.8V
+
+.. figure:: /images/peripheral_sensor_node_shield.png
+   :width: 350px
+   :align: center
+   :alt: Peripheral Sensor node shield
+
+   Peripheral Sensor node shield
+
+Pin assignment
+**************
+
+For the exact pin assignment, refer to the following table:
+
++-------------------------+------------+-----------------+
+| nRF54L15 connector pin  | SIGNAL     | Shield function |
++=========================+============+=================+
+| P1.13                   | BMI270 CS  | Chip Select     |
++-------------------------+------------+-----------------+
+| P2.10                   | ADXL362 CS | Chip Select     |
++-------------------------+------------+-----------------+
+| P2.08                   | SPI MOSI   | Serial Data In  |
++-------------------------+------------+-----------------+
+| P2.09                   | SPI MISO   | Serial Data Out |
++-------------------------+------------+-----------------+
+| P2.06                   | SPI SCK    | Serial Clock    |
++-------------------------+------------+-----------------+
+| P1.14                   | GPIO       | SPI Interrupt   |
++-------------------------+------------+-----------------+
+| P1.11                   | TWI SCL    | TWI Clock       |
++-------------------------+------------+-----------------+
+| P1.12                   | TWI SDA    | TWI Data        |
++-------------------------+------------+-----------------+
+
+Requirements
+************
+
+Shield is an add-on that you can attach to the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>` to extend its features and functionalities.
+
+.. figure:: /images/peripheral_sensor_node_assy.png
+   :width: 350px
+   :align: center
+   :alt: Peripheral Sensor node assembly
+
+   Peripheral Sensor node assembly
+
+The shield must be oriented in a way that allows its silkscreen to read the same way as PDK.
+The shield must not be rotated.
+
+Building and programming
+************************
+
+To build for the Peripheral Sensor node Shield, build for the compatible :ref:`nRF54L15 PDK <ug_nrf54l15_gs>` with the CMake ``SHIELD`` option set to ``pca63565``.
+See :ref:`cmake_options` for instructions on how to provide CMake options.
+
+You can use the following command:
+
+.. code-block:: console
+
+   west build -b nrf54l15pdk_nrf54l15_cpuapp -- -DSHIELD=pca63565
+
+If you use the |nRFVSC|, specify ``-DSHIELD=pca63565`` in the **Extra Cmake arguments** field when `setting up a build configuration <How to work with build configurations_>`_.
+
+Alternatively, add the shield in the project's :file:`CMakeLists.txt` file:
+
+.. code-block:: none
```

Comment:
Can we also add this article to the release notes? I think it can go under Documentation section (at the very end).
https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-changelog.html#documentation (release-notes-changelog.rst)
It could say:
* Added the ``:ref:`peripheral_sensor_node_shield` `` page. 

The information about the shield support should also be included via a PR with the relevant code.

Reply:
Done